central/centralhealth/service/service_impl.go

```diff
@@ -9,6 +9,7 @@ import (
 	"github.com/grpc-ecosystem/grpc-gateway/runtime"
 	"github.com/pkg/errors"
 	"github.com/stackrox/rox/central/role"
+	versionUtils "github.com/stackrox/rox/central/version/utils"
```

Comment:
This is seems to be flagging changes that were in https://github.com/stackrox/stackrox/pull/3346/files.  Not sure if you need a rebase or what so we don't have to weed through the stuff you recently merged.

Reply:
Yes, rebased.